### PR TITLE
Learning Team Agenda - January 14, 2021

### PR DESCRIPTION
**Pull request by ijlee2**
Please add comments or make edits if you want to add to the agenda! Core team and community members are all welcome to participate and add discussion points.

### CODE REVIEW

File: learning-team/2021-01/2021-01-14.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
```
Comment (ijlee2) [inline learning-team/2021-01/2021-01-14.md:37]: ### Super Rentals Refresh (IL, 15 min)

Mar, from Simplabs, has helped with (1) updating the design of and (2) improving UX for the [Super Rentals website](https://ember-super-rentals.netlify.app/).  The proposal document can be found below. Mar has asked for our feedback with 3 UX updates.

- https://www.notion.so/Super-Rentals-refresh-876e177ad8644562a00eb63a251a766f

File: learning-team/2021-01/2021-01-14.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### Super Rentals Refresh (IL, 15 min)
+
+Mar, from Simplabs, has helped with (1) updating the design of and (2) improving UX for the [Super Rentals website](https://ember-super-rentals.netlify.app/).  The proposal document can be found below. Mar has asked for our feedback with 3 UX updates.
+
+- https://www.notion.so/Super-Rentals-refresh-876e177ad8644562a00eb63a251a766f
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
```
Comment (ijlee2) [inline learning-team/2021-01/2021-01-14.md:42]:
```suggestion

### Maintaining ember-website (IL, 5 min)

A quick status update on changes that Isaac has made to [`ember-website` repo](https://github.com/ember-learn/ember-website/). Discuss near future plans.

```

File: learning-team/2021-01/2021-01-14.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### Super Rentals Refresh (IL, 15 min)
+
+Mar, from Simplabs, has helped with (1) updating the design of and (2) improving UX for the [Super Rentals website](https://ember-super-rentals.netlify.app/).  The proposal document can be found below. Mar has asked for our feedback with 3 UX updates.
```
Comment (ijlee2) [inline learning-team/2021-01/2021-01-14.md:39]:
```suggestion
Mar, from Simplabs, has helped with (1) updating the design of and (2) improving the user experience (UX) for the [Super Rentals website](https://ember-super-rentals.netlify.app/). The proposal document can be found below. Mar has asked for our feedback with 3 UX updates.
```

File: learning-team/2021-01/2021-01-14.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+### Super Rentals Refresh (IL, 15 min)
+
+Mar, from Simplabs, has helped with (1) updating the design of and (2) improving the user experience (UX) for the [Super Rentals website](https://ember-super-rentals.netlify.app/). The proposal document can be found below. Mar has asked for our feedback with 3 UX updates.
```
Comment (ijlee2) [inline learning-team/2021-01/2021-01-14.md:39]: Note to Learning Team:

Before the meeting, if you can quickly go through the [Super Rentals tutorial](https://guides.emberjs.com/release/tutorial/part-1/), I think that would help discussing suggestions and potential breaking changes.

**Review by jaredgalanis — APPROVED** [review]

**Comment by ijlee2** [comment]
@jenweber Thank you for facilitating the meeting today!

**Pull request merged**